plugins/amazonq/build.gradle.kts

```diff
@@ -51,3 +58,81 @@ tasks.check {
         }
     }
 }
+
+val downloadFlareManifest by tasks.registering(Download::class) {
+    src("https://aws-toolkit-language-servers.amazonaws.com/qAgenticChatServer/0/manifest.json")
```

Comment:
nit: assign to a const?

Reply:
we cannot share this with the plugin code